microsoft-edge/web-platform/web-platform.md

```diff
@@ -35,7 +35,7 @@ See [Site compatibility-impacting changes coming to Microsoft Edge](site-impacti
 <!-- ====================================================================== -->
 ## Move users to Microsoft Edge from Internet Explorer
 
-Many modern websites have designs that are incompatible with Internet Explorer.  When an Internet Explorer user visits an incompatible public website, the user may be instructed by the website that the website is incompatible with Internet Explorer, and that the user must switch to a more modern browser in order to use the website.
+ When an Internet Explorer user visits an incompatible public website, the user may be informed by the website that the website is incompatible with Internet Explorer, and that the user must switch to a more up-to-date browser in order to use the website properly.
```

Comment:
Unneeded whitespace at the beginning of this line.

Reply:
fixed in file